orchestration/hca_orchestration/tests/pipelines/test_pipelines.py

```diff
@@ -145,3 +162,27 @@ def test_cut_snapshot(*mocks):
     result = run_pipeline(job, config_name="test_create_snapshot.yaml")
 
     assert result.success
+
+
+def test_set_snapshot_public(*mocks):
```

Comment:
Maybe out of scope: I generally like to get negative tests for the sad path(s) as well. i.e., exercise the expectations if we get back more than snapshot from enumerate snapshot, what happens if jade 500s, etc. Given that this has already been exercised in dev I'm fine with it as-is but if this were to be longer lived I'd want that sort of coverage.

Reply:
I agree entirely and will add a ticket for this as well.
For the moment I am trying to keep the scope as narrow as possible since we are ultimately going to abandon this code, but if I have to touch it again, or if it starts to look like this solution may have to live longer and/or is buggy, I'll pick it up.

Reply:
[DI-277](https://broadworkbench.atlassian.net/browse/DI-277)